src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -157,7 +157,9 @@ const useCountStepTable = ({
       // on change function expects e.target.value for text fields,
       // in other cases it expects just the value
       const onChange = (e) => {
-        setValue(e?.target?.value ?? e);
+        const enteredValue = e?.target?.value ?? e;
+        table.options.meta?.updateData(cycleCountId, original.id, id, enteredValue);
+        setValue(enteredValue);
```

Comment:
Isn't it causing performance issues? We are updating the table data for each letter.